lib/gds_api/account_api.rb

```diff
@@ -78,6 +78,17 @@ def set_attributes(attributes:, govuk_account_session:)
     patch_json("#{endpoint}/api/attributes", { attributes: attributes }, auth_headers(govuk_account_session))
   end
 
+  # Look up the names of a user's attributes
+  #
+  # @param [String] attributes Names of the attributes to check
+  # @param [String] govuk_account_session Value of the session header
+  #
+  # @return [Hash] The attribute names (if present), and a new session header
+  def get_attributes_names(attributes_names:, govuk_account_session:)
```

Comment:
I think call the parameter `attributes`, as the two plurals in `attributes_names` feels weird to me

Reply:
What about `attribute_names`?

Reply:
`get_attributes` uses `attributes` for its parameter, which is a list of names; so for consistency I think this should too.